components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/cmd/FunctionCmd.java

```diff
@@ -0,0 +1,100 @@
+package org.wso2.apimgt.gateway.cli.cmd;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+import org.apache.commons.io.FileUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.wso2.apimgt.gateway.cli.exception.CLIRuntimeException;
+import org.wso2.apimgt.gateway.cli.utils.GatewayCmdUtils;
+import org.wso2.apimgt.gateway.cli.utils.RouteUtils;
+
+import java.io.File;
+import java.io.PrintStream;
+import java.util.List;
+
+@Parameters(commandNames = "function", commandDescription = "add function")
+public class FunctionCmd implements GatewayLauncherCmd {
+
+    private static final Logger logger = LoggerFactory.getLogger(FunctionCmd.class);
+    private static PrintStream outStream = System.out;
+
+    @Parameter(hidden = true, required = true)
+    private List<String> mainArgs;
+
+    @Parameter(names = {"-add", "--add function"}, hidden = true)
```

Comment:
Shorter term can be `-a` and I don't think the expanded term will work in this format, make it something like `--add`. No need of `function` prefix or suffix since the command is already `function`. So the user knows he/she is interacting with a function.
Please go through all below arguments and make sure its in the right format and following the standard argument terms

Reply:
modified the arguments